contracts/LiquidDemocracy.sol

```diff
@@ -0,0 +1,71 @@
+pragma solidity ^0.4.4;
+
+contract LiquidDemocracy {
+    ////////////////
+    // Data structures
+    ////////////////
+
+    // This is the type for a single option shown in a ballot
+    struct Proposal {
+        bytes32 name; // short name (up to 32 bytes)
+        uint voteCount; // number of accumulated votes
+    }
+
+    // A dynamically-sized array of `Proposal` structs.
+    Proposal[] public proposals;
+
+    // This is the type for a single voter metadata
+    struct Voter {
+        uint weight;
+        bool voted;
```

Comment:
this variable is probably unusable, given that there are many proposals, and you coud have voted or not for many

Reply:
This seems to be related to the conversation of how a proposal is defined. This simple, initial starting point is designed so that voters can vote once on the single proposal ballot (that has at least two options to choose from). Agree this is something we would have to modify in the future scenario of voters being able to vote on many ballots.

Reply:
ah I understand, it's just that there is also Proposal[] public proposals; so in the code it's implicit both that it's a single a multiple proposals at the same time :), let's deal with these issues in a next PR then